test-support/helpers/upload.js

```diff
@@ -1,19 +0,0 @@
-/*global triggerEvent, find, findWithAssert */
```

Comment:
I've moved this one into `addon-test-helpers` to allow for easy import

`import {upload} 'ember-file-upload/test-support'`

We probably don't need to keep backward compatibility in here as didn't work anyway.

Reply:
I think this worked for prior versions of ember (pre 3.0)

Reply:
Could you keep this, just in case? I know it may be broken, but I'd prefer not to major version bump.

Reply:
It's back.